core/mutator/entry/mutator.go

```diff
@@ -18,13 +18,16 @@ import (
 	"bytes"
 	"fmt"
 
-	"github.com/benlaurie/objecthash/go/objecthash"
-	"github.com/golang/glog"
-	"github.com/golang/protobuf/proto"
 	"github.com/google/keytransparency/core/crypto/signatures"
 	"github.com/google/keytransparency/core/mutator"
+
+	"github.com/google/trillian/crypto/keyspb"
 	"github.com/google/trillian/crypto/sigpb"
 
+	"github.com/benlaurie/objecthash/go/objecthash"
```

Comment:
Is this gofmt reordering of imports?

Reply:
The groups are manual. goimports keeps things in groups once I create the groups